engine/src/test/java/org/camunda/bpm/engine/test/api/authorization/ManagementAuthorizationTest.java

```diff
@@ -44,7 +44,7 @@
  */
 public class ManagementAuthorizationTest extends AuthorizationTest {
 
-  protected static final String REQUIRED_ADMIN_AUTH_EXCEPTION = "ENGINE-03029 Required admin authenticated group or user.";
+  protected static final String REQUIRED_ADMIN_AUTH_EXCEPTION = "Required admin authenticated group or user.";
```

Comment:
I think it would be nice if we could leave the original exception code if possible.

Reply:
But only for the original admin-only exception.

Reply:
Woah, you are fast. I wanted to add my reasoning behind it first :D
This exception is only used from a command context. So we usually have an exception message in the log like "ENGINE-16002 Exception while closing command context: ENGINE-03029 Required admin authenticated group or user."
I figured it's not necessary to have both codes.

Reply:
I assume the ENGINE-16002 is a very general code that occurs for a lot of different situations. Therefore, the special code ENGINE-03029 should still make sense.
We should probably also create one for the new error message with admin + permissions missing.